tasks/cnm-to-cma/tests/test_cnm2cma.py

```diff
@@ -0,0 +1,103 @@
+from unittest.mock import patch
+from cnm_to_cma import CNM2CMA
+import json
+from typing import List
+
+
+class TestCNM2CMA:
+    def setup_method(self):
+        with patch("src.cnm2cma.cnm_to_cma.CNM2CMA.__init__", return_value=None):
+            self.cnm2cma = CNM2CMA()
+
+    def test_get_input_files(self):
+        product = {"files": [{"name": "file1"}, {"name": "file2"}]}
+        result = self.cnm2cma.get_input_files(product)
+        assert result == [{"name": "file1"}, {"name": "file2"}]
+
+        product = {
+            "filegroups": [
+                {"files": [{"name": "file3"}]},
+                {"files": [{"name": "file4"}]},
+            ]
+        }
+        result = self.cnm2cma.get_input_files(product)
+        assert result == [{"name": "file3"}, {"name": "file4"}]
```

Comment:
There should be a test that shows an error and other use cases for the `get_input_files` method like:

- What is the expected behaviour if both "files" and "filegroups" are empty?
- What is the expected behaviour if they are both populated?